pkg/cloudprovider/cloudprovider.go

```diff
@@ -94,11 +94,7 @@ func (c *CloudProvider) Create(ctx context.Context, nodeClaim *karpv1.NodeClaim)
 		return nil, cloudprovider.NewNodeClassNotReadyError(fmt.Errorf("EC2NodeClass %q is incompatible with Karpenter v1, specify your Ubuntu AMIs in your AMISelectorTerms", nodeClass.Name))
 	}
 	// TODO: Remove this after v1
```

Comment:
nit: still relevant?

Reply:
Yeah, it's still relevant since that comment should technically remain on the release-v1.0.x branch. You're right though that we've already removed it :P